.github/workflows/integration-tests.yml

```diff
@@ -1,27 +1,27 @@
-name: Next.js Integration test
+name: Integration test
 
 on:
   pull_request:
-    branches: [main, 'chore/ci-e2e-nextjs']
+    branches: [main, 'chore/ci-integration-tests']
     # Run on release PRs only, or changes to this same file
-    paths: ['CHANGELOG.md', '.github/workflows/e2e-nextjs.yml']
+    paths: ['CHANGELOG.md', '.github/workflows/integration-tests.yml']
 
 jobs:
   integration:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        bundler: ['default', 'esbuild']
+        site: ['netlify/netlify-plugin-nextjs']
```

Comment:
I tried adding https://github.com/netlify/netlify-plugin-gatsby, but it failed to build.